linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -9,7 +9,8 @@
    "source": [
```

Comment:
<p>We should probably modify the SSA simulation to include a lot of these observations when we do our next big enhancement to the sim</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>